internal/proto/query_coord.proto

```diff
@@ -547,6 +548,7 @@ enum SyncType {
   Remove = 0;
   Set = 1;
   Amend = 2;
+  UpdateVersion = 3;
```

Comment:
UpdateVersion is a complex operation, which is different from the atomic operations of remove, set and amend mentioned above.

Reply:
UpdateVersion just like remove and set, will modify the segment distribution on shard delegator